strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/artifact/MavenArtifactUtils.java

```diff
@@ -1,76 +1,93 @@
 package org.carlspring.strongbox.artifact;
 
-import org.carlspring.maven.commons.util.ArtifactUtils;
 import org.carlspring.strongbox.providers.io.RepositoryFiles;
 import org.carlspring.strongbox.providers.io.RepositoryPath;
 
 import java.io.IOException;
 
+import org.apache.commons.lang3.StringUtils;
 import org.apache.maven.artifact.Artifact;
+import org.apache.maven.index.artifact.Gav;
+import org.apache.maven.index.artifact.M2GavCalculator;
 import org.javatuples.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * @author Przemyslaw Fusik
  */
 public class MavenArtifactUtils
 {
 
-    public static String convertArtifactToPath(Artifact artifact)
-    {
-        return ArtifactUtils.convertArtifactToPath(artifact);
-    }
-
-    public static String getArtifactFileName(MavenArtifact artifact)
-    {
-        return ArtifactUtils.getArtifactFileName(artifact);
-    }
+    private static final Logger logger = LoggerFactory.getLogger(MavenArtifactUtils.class);
 
-    public static Pair<String, String> getArtifactGroupId(RepositoryPath repositoryPath)
-        throws IOException
-    {
-        MavenArtifact tmpArtifact = MavenArtifactUtils.convertPathToArtifact(repositoryPath);
+    private static final M2GavCalculator M2_GAV_CALCULATOR = new M2GavCalculator();
 
-        return Pair.with(tmpArtifact.getGroupId(), tmpArtifact.getArtifactId());
-    }
-    
-    public static MavenArtifact convertPathToArtifact(RepositoryPath repositoryPath) throws IOException
+    public static String convertArtifactToPath(Artifact artifact)
     {
-        String path = RepositoryFiles.relativizePath(repositoryPath);
-        Artifact artifact = ArtifactUtils.convertPathToArtifact(path);
-        
-        return new MavenRepositoryArtifact(artifact, repositoryPath);
+        final Gav gav = new Gav(artifact.getGroupId(),
```

Comment:
we probably better to make `MavenArtifactUtils` method for this.

Reply:
ok, will do